com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraEditor.Handlers.cs

```diff
@@ -14,9 +14,10 @@ void OnSceneGUI()
             if (HDRenderPipeline.currentPipeline == null)
                 return;
 
-            var c = (Camera)target;
+            if (!(target is Camera c) || c == null)
```

Comment:
I'm curious: `target is Camera c` if target is null, c is (Camera)null right? Then why do we need the null check?

Reply:
because is null on the c++ world, not in c# the `is` operator is not overriden